lib/Runtime/Library/JavascriptBigInt.cpp

```diff
@@ -11,6 +11,26 @@ namespace Js
         return RecyclerNew(scriptContext->GetRecycler(), JavascriptBigInt, content, cchUseLength, isNegative, scriptContext->GetLibrary()->GetBigIntTypeStatic());
     }
 
+    JavascriptBigInt * JavascriptBigInt::CloneToScriptContext(Var aValue, ScriptContext * scriptContext)
+    {
+        return JavascriptBigInt::New(UnsafeVarTo<JavascriptBigInt>(aValue), scriptContext);
+    }
+
+    JavascriptBigInt * JavascriptBigInt::New(JavascriptBigInt * pbi, ScriptContext * scriptContext)
+    {
+        JavascriptBigInt * bigintNew = RecyclerNew(scriptContext->GetRecycler(), JavascriptBigInt, scriptContext->GetLibrary()->GetBigIntTypeStatic());
+        bigintNew->m_length = pbi->m_length;
+        bigintNew->m_maxLength = pbi->m_maxLength;
+        bigintNew->m_isNegative = pbi->m_isNegative;
+        bigintNew->m_digits = bigintNew->m_reservedDigitsInit;
+        bigintNew->Resize(bigintNew->m_length);
+        for (digit_t i = 0; i < bigintNew->m_length; i++)
```

Comment:
nit: could use memcpy for this step (the compiler might be clever enough to emit something similar based on this loop, I'm not sure)

Reply:
fixed